lib/kotkowo_web/live/news_live/announcements/news_article.html.heex

```diff
@@ -7,34 +7,64 @@
       Z ostatniej chwili
     </:crumb>
     <:crumb navigate={~p"/aktualnosci/z-ostatniej-chwili/#{@article_id}"} active>
-      <%= @article.title %>
+      <span :if={@article}>
+        <%= @article.title %>
+      </span>
+      <span :if={@article == nil}>
+        <%= @article_id %>
+      </span>
     </:crumb>
   </.breadcrumb>
   <div class="flex flex-col lg:flex-row justify-between px-5">
-    <div class="flex flex-col flex-wrap w-full lg:w-[865px]">
-      <h1 class="w-full lg:w-[832px] text-primary text-center lg:text-justify lg:mt-0 font-manrope font-extrabold xl:text-4xl text-3xl">
+    <div :if={!@article} class="flex gap-y-10 flex-col flex-wrap" }>
+      <div class="w-full lg:min-w-[832px] bg-gray-300 animate-pulse lg:h-12 h-16 mt-2 rounded">
+      </div>
+      <div class="h-[340px] bg-gray-300 animate-pulse rounded-2xl"></div>
+      <div class="flex flex-col gap-y-4">
+        <div class="bg-gray-300 animate-pulse w-2/3 h-8 rounded"></div>
+        <div :for={_ <- 0..3} class="bg-gray-300 animate-pulse even:w-4/5 h-6 rounded"></div>
+      </div>
+    </div>
+    <div :if={@article} class="flex flex-col flex-wrap w-full lg:w-[865px]">
+      <h1 class="w-full lg:w-[832px] text-primary text-center lg:text-justify lg:mt-0 font-manrope font-extrabold xl:text-4xl text-3xl mt-2 lg:mt-0">
         <%= @article.title %>
       </h1>
       <img
         class="w-full h-[340px] object-cover  rounded-2xl mt-10"
-        src={GalleryImage.url(@article.image)}
+        src={@article.image.url}
         alt="article image"
       />
       <span class="text-2xl font-semibold leading-10 mt-10"><%= @article.introduction %></span>
-      <span class="text-xl leading-10 w-full lg:w-[865px] overflow-auto break-words">
-        <%= @article.content |> HtmlSanitizeEx.markdown_html() |> Phoenix.HTML.raw() %>
-      </span>
+      <div
+        id="article-content"
+        class="text-xl leading-10 w-full lg:w-[865px] overflow-auto"
+        article-content={@article.content}
+        phx-hook="sanitize_article_html"
+      >
+        <%= @content |> Phoenix.HTML.raw() %>
```

Comment:
this is not safe I think. even if the "sanitize_article_html" hook sanitizes the content gets rendered already and malicious code might be injected.
pass the content through the socket or if possible through a `data-` attribute.